decidim-verifications/app/views/layouts/decidim/authorizations.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render "layouts/decidim/application" do %>
```

Comment:
Note that we still have the landmark WCAG issue on this page, please check the a11y toolbar for this:
![Landmark a11y error](https://user-images.githubusercontent.com/864340/203937381-7f6ae38d-2e6c-4081-b0a3-2c40c8471c37.png)

Reply:
Moved to a new issue